### ISSUE TITLE
[Bug]: URLs don't display as links

### ISSUE THREAD

**Issue by MacChuck** [FTBTeam/FTB-Modpack-Issues#885]
### Modpack

Direwolf 20 1.18

### Modpack version

1.3.1

### Describe the bug

Updated to v1.3.1 of the pack. URLs in chat, including blockshot URLs, don't appear as clickable links. Tested in single player and multiplayer. Tried toggling chat URLs on and off in client settings.

### Steps to reproduce

Copy and paste a url into chat.

### Expected behaviour

URLs should turn into clickable links.

**Comment by Sunekaer** [comment]
Did this work before 1.3.1?

**Comment by MacChuck** [comment]
Yep, URLs in chat in 1.2 of the pack showed up as clickable links and worked.

**Comment by Sunekaer** [comment]
Hmm okay ill look in to it.

**Comment by phit** [comment]
I suspect this may be related to https://github.com/FTBTeam/FTB-Ranks/commit/9e2c8957e67c9d309c7395f36efd9afc21268921
